.changeset/rich-parks-sin.md

```diff
@@ -0,0 +1,18 @@
+---
+"@palantir/pack.document-schema.model-types": patch
+"@palantir/pack.sdkgen.demo-template": patch
+"@palantir/pack.sdkgen.pack-template": patch
+"@palantir/pack.document-schema.type-gen": patch
+"@palantir/pack.state.foundry-event": patch
+"@palantir/pack.sdkgen": patch
+"@palantir/pack.state.foundry": patch
+"@palantir/pack.auth.foundry": patch
+"@palantir/pack.state.react": patch
+"@palantir/pack.state.core": patch
+"@palantir/pack.auth": patch
+"@palantir/pack.schema": patch
+"@palantir/pack.core": patch
+"@palantir/pack.app": patch
+---
+
+Bump dependencies
```

Comment:
I'm on the fence about requiring a changeset here, ideally changesets should enumerate the differences when it does a release, I don't think it does from what I can see - I guess it still needs a decision on whether deps are patch or minor or major changes